streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/rule/sql/SqlScript.java

```diff
@@ -142,7 +154,7 @@ public StreamlineEvent apply(Values values) {
                         return valuesConverter.convert(values, inputEvent);
                     }
                 });
-            } else {
+            } else if (result instanceof StreamlineEvent){
```

Comment:
if result is null we return an empty list. Inside `if (result != null)` can throw an exception in a final `else`